automation.py

```diff
@@ -108,6 +108,8 @@ def getOr(name):
             self.runOnProviders(lambda x: x.update_config(additional_config))
             # Step 7
             self.__dict__.update(self.provider_dictionary)
+            # And check the database
+            self.dbProvider.check_database()
```

Comment:
`TestConfigDatabaseProvider` doesn't have a `check_database` method so this throws an error